### PR TITLE
Adit mode

### PR DESCRIPTION
**Pull request by Dianakove32**

### CODE REVIEW

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+
+const Card = (props: IProps) => {
+const {data,removeCardHandler, editCardHandler,isEdit } = props
 
     return (
         <div className="cards-container">
-        {data.map((el: IProps) => (
+        {data.map((el: IPropsData) => (
             <div className="card" key={el.id}>
+            {isEdit ? <span id={el.id} onClick = {()=>removeCardHandler(el.id) }>&#10060;</span> : null}
```
Comment (Fancy322vav) [inline src/components/Card/Card.tsx]: This should be a global btn, like turn off 'edit mode'. Not for each card.

File: src/App.tsx [hunk 2]
```diff
@@ -33,23 +33,50 @@ let initialState: CardsDataDTO[] = [
...
+        let filteredData = data.filter((el) => el.id !== id);
+
+        setData(filteredData);
+    };
+
+    const editCardHandler = (id: string) => {
+        setIsEdit(true);
+    };
+
+    const openModal = () => {
```
Comment (Fancy322vav) [inline src/App.tsx]: This should be one method like `toggleModal` based on `prevState`.

**Review by Fancy322vav — APPROVED** [review]
Good refactoring. Left some changes for next PR. This one is Approved!

File: src/components/Card/Card.tsx (hunk 1 above)
Comment (Dianakove32) [reply src/components/Card/Card.tsx]: done

File: src/App.tsx (hunk 2 above)
Comment (Dianakove32) [reply src/App.tsx]: done

File: src/components/Card/Card.tsx [hunk 3]
```diff
@@ -1,23 +1,87 @@
...
+    const saveCardHandler = (event: { preventDefault: () => void; }) => {
+        event.preventDefault()
+        editCardHandler();
+    };
 
     return (
+        <div className="content-container">
+        <div >
+            <button className="btn-add" onClick={editCardHandler}>
+            Edit
```
Comment (Fancy322vav) [inline src/components/Card/Card.tsx]: Here should be btn 'Cancel' in the edit mode. It should clear all the done changes.

**Review by Fancy322vav — APPROVED** [review]
Looks great. Approved. Only some changes need to be done before next PR.
Please add a validation to prevent from leaving empty fields in cards while editing.

File: src/components/Card/Card.tsx (hunk 3 above)
Comment (Dianakove32) [reply src/components/Card/Card.tsx]: done

**Pull request merged**